package.json

```diff
@@ -28,11 +28,13 @@
     "async": "1.5.x",
     "code": "3.x.x",
     "hapi": "10.x.x",
-    "lab": "10.x.x"
+    "lab": "10.x.x",
+    "npmignore": "0.2.x"
   },
   "scripts": {
     "test": "lab -m 5000 -t 100 -v -La code",
-    "test-cov-html": "lab -m 5000 -r html -o coverage.html -La code"
+    "test-cov-html": "lab -m 5000 -r html -o coverage.html -La code",
+    "update-npmignore": "npmignore -i examples,images,test,API.md,CONTRIBUTING.md"
```

Comment:
I guess we can just close this pull since [gitignore](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package) will suffice for this purpose.